### PR TITLE
Added a new feature

### PR DESCRIPTION
**Pull request by jaxmmmm**
This feature is for adding an autocomplete into the search box.

### CODE REVIEW

File: lab-exercise-1/PathuFeature.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+public class <Yourname>Feature {
+      public void feature() {
```
Comment (nekongum) [inline lab-exercise-1/PathuFeature.java:2]: change your class name pls

File: lab-exercise-1/PathuFeature.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-public class <Yourname>Feature {
+public class PathuFeature {
```
Comment (nekongum) [inline lab-exercise-1/PathuFeature.java:1]: good

File: lab-exercise-1/PathuFeature.java (hunk 1 above)
Comment (Napassawxn) [inline lab-exercise-1/PathuFeature.java:2]: I think you should fix it

Comment (Napassawxn) [reply lab-exercise-1/PathuFeature.java:2]: I think you should fix it

File: lab-exercise-1/PathuFeature.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+      public void feature() {
+          // this is a new feature
```
Comment (JNapat) [inline lab-exercise-1/PathuFeature.java:3]: empty?

File: lab-exercise-1/PathuFeature.java (hunk 1 above)
Comment (6588041chalisakengkaewpennapa) [reply lab-exercise-1/PathuFeature.java:2]: I think so this is good

**Comment by rmabow03** [comment]
it not complete the feature, can you add the feature in your code and change you name class.

File: lab-exercise-1/PathuFeature.java (hunk 1 above)
Comment (rmabow03) [reply lab-exercise-1/PathuFeature.java:2]: change the name

File: lab-exercise-1/PathuFeature.java (hunk 3 above)
Comment (rmabow03) [reply lab-exercise-1/PathuFeature.java:3]: don't forget your feature

File: lab-exercise-1/PathuFeature.java [hunk 4]
```diff
@@ -0,0 +1,5 @@
+public class <Yourname>Feature {
+      public void feature() {
+          // this is a new feature
```
Comment (Napassawxn) [inline lab-exercise-1/PathuFeature.java:3]: Delete it

File: lab-exercise-1/PathuFeature.java [hunk 5]
```diff
@@ -1,5 +1,8 @@
 public class PathuFeature {
       public void feature() {
           // this is a new feature
+          int x = 0;
+          x = x+1;
```
Comment (nekongum) [inline lab-exercise-1/PathuFeature.java:5]: good.

File: lab-exercise-1/PathuFeature.java [hunk 6]
```diff
@@ -1,4 +1,4 @@
-public class <Yourname>Feature {
```
Comment (Napassawxn) [inline lab-exercise-1/PathuFeature.java]: Dont forget to change name

File: lab-exercise-1/PathuFeature.java (hunk 5 above)
Comment (rmabow03) [reply lab-exercise-1/PathuFeature.java:5]: yessss

**Pull request merged**